web/modules/custom/portland/modules/portland_location_picker/css/portland_location_picker.css

```diff
@@ -0,0 +1,14 @@
+/**
+ * @file
+ * Webform example composite styles.
+ */
+
+.webform_example_composite_multiple-table--first_name {}
```

Comment:
@jookyg , there are lots of references to these name, birth, and gender fields in this CSS as well as in this file `web/modules/custom/portland/modules/portland_location_picker/src/Plugin/WebformElement/PortlandLocationPicker.php`. They don't seem to be related to location? If they are just examples, do you want to add some notes in your followup story to remove them?